README.md

```diff
@@ -1,16 +1,25 @@
 # DevTools RFCs
 
-Some changes to DevTools need feedback from the team and community to reach a consensus. In such a case you should use this repository to discuss the change proposal.
+Some changes to DevTools need feedback from the team and community to reach a consensus. In such cases we should use the RFC process to discuss the proposed changes.
 
-The DevTools RFC process aims to be lightweight and focused on conversations. Therefore opening a DevTools RFC is as easy as opening an issue in this repository to discuss the proposal.
+We aim for the process to be lightweight and focused on conversations. Therefore opening a DevTools RFC is as easy as opening an issue in this repository.
 
-The [DevTools RFC team][] will be ultimately responsible for accepting or rejecting a change proposal, but the discussion is open to everybody.
+The [DevTools RFC team][] will be ultimately responsible for accepting or rejecting proposals, but the discussion is open to everybody.
 
-The list of RFCs currently under discussion can be found at: [Active RFCs][].
+The list of RFCs currently under discussion can be found at [Active RFCs][].
+
+## What kind of changes should be discussed?
+
+We aim to discuss specific and actionable changes, i.e. we aim to answer questions about work you're doing or about to do either on the DevTools codebase or on something that *interacts* with DevTools. For example:
+
+* you are about to make a big change that affects a number of panels and would like feedback on the solution you came up with.
+* you are about to make a breaking change on an essential API we interact with, and would like to discuss it and get feedback from us beforehand.
+
+This distinction can be unclear at times, but we will try our best to discuss proposals, taking into account our own time limitations.
```

Comment:
Is `about work you're doing or about to do` in middle of this paragraph clear enough?
If you just read that sentence, sure it is! But the examples that follow do not necessarily highlight that particular fact.

I imagine that's clear enough for the RFC team, who has to clearly understand that. We shouldn't make it hard to suggest RFC, but we should be careful while receiving them and reject them early if that are not resource-actionable.

Reply:
I thought "you are about to make a big change" or "a breaking change" reflects "work you're doing or about to do" -- how would you rephrase it?

Reply:
I've updated the paragraph, please let me know if it still doesn't look clear, @ochameau - I'm going to resolve this conversation for now.